test/script_tools.py

```diff
@@ -43,6 +41,16 @@ class TestExtendedSimpleOption(TestCase):
     """
     Tests for the ExtendedSimpleOption class.
     """
+    def setUp(self):
+        """Backup sys.argv"""
+        super(TestCase, self).setUp()
+        self._old_argv = sys.argv
+        sys.argv = sys.argv[:1]
```

Comment:
this really fixes the tests this time (see pr title)
If you run the tests with `python setup.py test --xmlrunner=xxx` later on the tests for ExtendedSimpleOption will fail with `No such option --xmlrunner` 
see https://jenkins1.ugent.be/job/vsc-utils/PYTHON=CPython-2.7/11/console